source/includes/_bulk_data.md

```diff
@@ -54,9 +54,9 @@ The core steps to get this up and running are:
 4. Setting up your `aws-lambda-redshift-loader` lambda.
 5. Configure your bulk data webhook in the Control Shift settings console.
 
-**Note:** All resources should be set up in the same AWS region (ex: us-east-1) so they can access each other. "US Standard" and "us-east-1" are the same thing.
+**Note:** All resources should be set up in the same AWS region (i.e.: us-east-1) so they can access each other. "US Standard" and "us-east-1" are the same thing.
```

Comment:
should use "e.g.", not "i.e.", here.

(Latin Nerd sidebar: "i.e." == "id est" ~= "that is", while "e.g." == "exempli gratia" ~= "for example")